src/hooks/createResolvers/index.js

```diff
@@ -35,22 +35,16 @@ module.exports = (
             fallbackLocales: { type: '[String!]' },
           },
           resolve: async (parent, args, context, info) => {
-            if (context.sourceDatocms.localeState) {
-              context.sourceDatocms.localeState.clear(info);
-            }
-            if (context.sourceDatocms.fallbackLocales) {
-              context.sourceDatocms.fallbackLocales.clear(info);
-            }
```

Comment:
This was a code smell that something is not right - if something has to clear data for current `info.path` which should be first time we visit it - the setup seemed problematic overall.

Now with fresh "locale context" it should be just not possible for those things to have to be cleared as we should enter this code path for given `info.path` for the first time every time